src/main/java/com/github/manolo8/darkbot/modules/utils/NpcAttacker.java

```diff
@@ -144,9 +144,11 @@ private boolean shouldSab() {
 
     private boolean shouldRsb() {
         if (!main.config.LOOT.RSB.ENABLED || !target.npcInfo.extra.has(NpcExtra.USE_RSB)) return false;
-        if (usedRsb < System.currentTimeMillis() - main.config.LOOT.RSB.AMMO_REFRESH) usedRsb = System.currentTimeMillis();
+        boolean isReady = bar.findItemById("ammunition_laser_rsb-75").map(i -> i.activatable).orElse(false);
```

Comment:
This breaks support for any other burst-based ammo (eg: the x7 ammo), should keep that into consideration and search for the item that is actually being selected by the key

Reply:
Wrong, all rapid salvo batteries share same cooldown, like regular how rockets do.